src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -121,12 +121,10 @@ public async Task<ResourceDownloadResults> EnsureResourceAsync(IHostEnvironment
             for (int i = 0; i < retryTimes; ++i)
             {
                 var thisDownloadResult = await DownloadFromUrlAsync(env, ch, url, fileName, timeout, filePath);
-
                 if (string.IsNullOrEmpty(thisDownloadResult))
```

Comment:
What was the point of deleting the previous line? And the line below? Please try to keep your PRs clean.